tests/e2e/settings-view/specs/devtools-1.ts

```diff
@@ -0,0 +1,36 @@
+import { expect } from "@wdio/globals";
+import { VSCodePO } from "../../page-objects";
+
+describe("Settings view", () => {
+    describe("devtools", () => {
+        it("should fail if option is not enabled", async () => {
```

Comment:
In `samples/settings-view` option `devtools` is not specified. So this test should fail because grid is not run.